src/components/SubStepForms/FullNameStep.tsx

```diff
@@ -41,7 +41,14 @@ type FullNameStepProps<TFormID extends keyof OnyxFormValuesMapping> = SubStepPro
         lastName: string;
     };
 
+    /** Should show the help link or not */
     shouldShowHelpLinks?: boolean;
+
+    /** Custom label of the first input  */
+    customFirstNameLabel?: string;
+
+    /** Custom label of the second input */
```

Comment:
Similarly can we do this?

```suggestion
    /** Custom label of the last name input */
```